src/website/src/app/app.component.ts

```diff
@@ -45,6 +45,21 @@ export class AppComponent implements OnInit {
     });
   }
 
+  trackSurveyAction(eventLabel: string) {
+    if (window.trackSurveyAlert) {
+      window.trackSurveyAlert(eventLabel);
+    }
+
+    // close survey in future visits
+    if (eventLabel === 'close-survey') {
+      localStorage.setItem('closeSurvey', JSON.stringify(true));
```

Comment:
I would prefer to store this in a cookie, localStorage lives _forever_.

Reply:
Updated to use a cookie that expires in 60 days. That should be long enought for this survey.